mix.exs

```diff
@@ -1,11 +1,26 @@
 defmodule Trento.MixProject do
   use Mix.Project
 
+  @version "1.1.0"
+  defp get_version do
+    case System.get_env("VERSION") do
+      nil -> get_version_from_git()
+      version -> version
+    end
+  end
+
+  defp get_version_from_git do
+    case File.cwd!() |> Path.join("hack/get_version_from_git.sh") |> System.cmd([]) do
```

Comment:
I am wondering whether this is necessary here.
What I mean is that we're setting the `VERSION` env variable at build time by using `get_version_from_git.sh` so, do we really need to do that here? :thinking: 

The alternative I see would be: either at build time the `VERSION` has been set, or we fallback to the default version.

Am I missing something? would that work?

Reply:
Yes, I guess you are right. I just implemented this way to have the way to debug it in my development environment. 
We can delete this case, no issues with it.

Ok, I will remove it for the sake of simplicity

Reply:
Edit: Ok, now I know why I have added it...
It was for the community version. The `VERSION` replacement only affect the suse container, but the community doesn't have such a replacement or Dockerfile edition, and the `docker/build-push-action@v3` job doesn't make it that straightforward

Reply:
Oh I see, is that the reason why e2e tests are failing?

Anyway, no strong opinion from my side. It is just fine like you did. :muscle:

Reply:
> and the docker/build-push-action@v3 job doesn't make it that straightforward.
ah I see, have to say I am not a big fan of executing shell scripts from elixir but if there is not another way i am ok with it